src/Microsoft.AspNetCore.Mvc.Core/Controllers/IControllerArgumentBinder.cs

```diff
@@ -9,15 +9,20 @@ namespace Microsoft.AspNetCore.Mvc.Controllers
     /// <summary>
     /// Provides a dictionary of action arguments.
     /// </summary>
-    public interface IControllerActionArgumentBinder
+    public interface IControllerArgumentBinder
```

Comment:
Discussed with @Eilon and we're still making changes in this area until we're confident that we have it right.